jaxley/modules/base.py

```diff
@@ -1762,6 +1810,25 @@ def insert(self, channel: Channel):
         for key in channel.channel_states:
             self.base.nodes.loc[self._nodes_in_view, key] = channel.channel_states[key]
 
+    @only_allow_module
+    def diffuse(self, state: str):
```

Comment:
We might want to add a method `delete_pumps` or the like. Also fine with adding it later and opening an issue for now.

Reply:
I agree. I changed `.delete_channels()` to `.delete()` and made it also handle `Pump`s. I also added `.delete_diffusion()`.